### PR TITLE
Piyush_Assignment_20_1.5Hour

### PR DESCRIPTION
**Pull request by PiyushPJun20**
[Screenshot.docx](https://github.com/KrishnaKTechnocredits/JAVATechnoJun20/files/4844897/Screenshot.docx)

### CODE REVIEW

File: src/DuplicateNum.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+public class DuplicateNum {
+
+	void displayDuplicate(int[] intArr) {
+
+		System.out.print(" Duplicated Number is : ");
+		for (int i = 0; i < intArr.length; i++) {
+			for (int j = i + 1; j < intArr.length; j++) {
+				if (intArr[i] == intArr[j]) {
+					System.out.println(intArr[i]);
+
```
Comment (ShrutiDJun20) [inline src/DuplicateNum.java:10]: Just a suggestion here: we should also handle the scenario where what should happen if there is no duplicate in the array. But no need to change it in the code as your code is as per requirement..

**Review by ShrutiDJun20 — APPROVED** [review]

File: src/DuplicateNum.java (hunk 1 above)
Comment (KrishnaKTechnocredits) [reply src/DuplicateNum.java:10]: @ShrutiDJun20 you are correct ! Ideally we should include that.

File: src/Armstrong.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
+public class Armstrong {
+	void ArmstrongNum(int num) {
+
+		int aNum = 0, temp = num;
+		while (temp > 0) {
+			aNum += ((temp % 10) * (temp % 10) * (temp % 10));
```
Comment (KrishnaKTechnocredits) [inline src/Armstrong.java:7]: as an alternative, we can use Math.pow(temp%10,3) function when

File: src/MissingNumber.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
+public class MissingNumber {
+
+	void displayMissingNum(int[] intArr) {
+		
+		boolean flag;
+		System.out.println(" Missing Numbers are : ");
+		for (int i=1; i<=10; i++) {
```
Comment (KrishnaKTechnocredits) [inline src/MissingNumber.java:7]: Please use index instead of i, it will be good during interview as well as in regular practice.

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me. Keep it up.

**Pull request merged**